src/app/code/community/Allopass/Hipay/Helper/Data.php

```diff
@@ -616,6 +616,17 @@ public function getCheckoutSuccessPage($payment) {
 		return is_null(Mage::getStoreConfig('payment/'.$payment->getMethod().'/success_redirect_page')) ?
 			'checkout/onepage/success' :
 			Mage::getStoreConfig('payment/'.$payment->getMethod().'/success_redirect_page');
+	}<
+
+	/**
+	 * @param Mage_Sales_Model_Order_Payment $payment
+	 *
+	 * @return string
+	 */
+	public function getCheckoutFailurePage($payment) {
+		return is_null(Mage::getStoreConfig('payment/'.$payment->getMethod().'/failure_redirect_page')) ?
```

Comment:
The condition is_null() don't work, we have a problem in the sendRequest(). Prefer the function empty().